ft_volumenormalise.m

```diff
@@ -97,6 +101,15 @@
 % check if the input data is valid for this function
 mri = ft_checkdata(mri, 'datatype', 'volume', 'feedback', 'yes', 'hasunit', 'yes', 'hascoordsys', 'yes');
 
+% Check coordsys consistency
```

Comment:
I think that it is more logical to move this chunk (or at least a chunk of code that has an equivalent outcome) to after the definition of the cfg-options. Right now, it can happen that a user specifies cfg.template to be 'T1.nii' by hand, in which case an error is thrown, while this shouldn't be necessary (since a default 'spm' templatecoordsys would then suffice)